db/sql/table_partial.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS partial;
+CREATE TABLE partial
+(
+    partial_id CHAR(36)     NOT NULL COMMENT 'UUID to identify a single unique version',
+    doc_type   VARCHAR(255) NOT NULL COMMENT 'user provided document type, works as a category',
```

Comment:
Why is doc_type here?  In the past, to identify a partial in the template template table, the doc_type was "template/" and then a string to identify the type of partial (i.e. partial, image...).

Reply:
see other comment